tooling/api/src/fs.ts

```diff
@@ -550,6 +551,31 @@ async function renameFile(
   })
 }
 
+/**
+ * Check if a path exists.
+ * @since 1.1.0
+ * @example
+ * ```typescript
+ * import { exists, BaseDirectory } from '@tauri-apps/api/fs';
+ * // Check if the `$APPDIR/avatar.png` file exists
+ * await exists('avatar.png', { dir: BaseDirectory.App });
+ * ```
+ *
+ * @param path A path of the file or directory to check.
+ * @param options Configuration object.
+ * @returns A promise indicating the success or failure of the operation.
```

Comment:
```suggestion
 * @example
 * ```typescript
 * import { exists, BaseDirectory } from '@tauri-apps/api/fs';
 * // Check if the `$APPDIR/avatar.png` file exists
 * await exists('avatar.png', { dir: BaseDirectory.App });
 * ```
 * @since 1.1.0
```